conf/locale/he/LC_MESSAGES/django.po

```diff
@@ -14167,8 +14178,9 @@ msgid "Additional Resources"
 msgstr "משאבים נוספים"
 
 #: lms/templates/courseware/course_about.html
+msgctxt "self"
 msgid "enroll"
-msgstr "הירשם"
+msgstr "רישום אוטומטי"
 
```

Comment:
I add this here although the line is not included in the technical doc. Line 334 in lms/templates/courseware/course_about.html

Reply:
this makes sense